docs/en_US/_templates/index.html

```diff
@@ -5,6 +5,7 @@
 {% block document %}
 
 <div>
+  <div class="chinese"><a href="https://github.com/microsoft/nni/blob/master/README_zh_CN.md">简体中文</a></div>
```

Comment:
It's great to have this link. Can it link to same place like en in readthedocs? for example, `https://nni.readthedocs.io/zh/latest/`?

Reply:
see https://github.com/microsoft/nni/pull/2483/commits/69a1c3212268c899897b349584402af293a1f21a